css-align-3/content-distribution/place-content-shorthand-004.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <title>CSS Box Alignment: place-content shorthand - Invalid values</title>
+      <link rel="author" title="Javier Fernandez" href="mailto:jfernandez@igalia.com" />
+      <link rel="help" href="http://www.w3.org/TR/css3-align/#propdef-place-content" />
+      <meta name="flags" content="invalid" />
```

Comment:
I believe this flag is not needed. It's meant when a test itself has some issues, for example if you don't close a tag for whatever reason, as you're testing that in purpose. But your tests are syntactically valid, you're just checking that some invalids values don't work. So please remove this line.

BTW, note that most of the flags are somehow deprecated, and only a few will remain in the future.